### PR TITLE
created node list

### PR DESCRIPTION
**Pull request by vinperdom**

### CODE REVIEW

File: src/parser.l [hunk 1]
```diff
@@ -2,9 +2,17 @@
...
 #include "parser.tab.h"
 
+
 void yyerror(char *);
 %}
 
+
+/* RegEx */
+number ([0-9])+
+var ([_a-zA-Z][_a-zA-Z0-9])*
```
Comment (fmenezes) [inline src/parser.l:12]: How about calling it identifier?

Comment (vinperdom) [reply src/parser.l]: Done!

File: src/parser.l [hunk 2]
```diff
@@ -8,18 +8,19 @@ void yyerror(char *);
 
 
 /* RegEx */
-number ([0-9])+
-var ([_a-zA-Z][_a-zA-Z0-9])*
-
+number [0-9]+
+string [_a-zA-Z]
+identifier {string}[_a-zA-Z0-9]*
+var "var"
```
Comment (fmenezes) [inline src/parser.l:14]: remove this line

Comment (vinperdom) [reply src/parser.l]: Done!

**Review by fmenezes — APPROVED** [review]
good job 👍

**Pull request merged**